.appveyor.yml

```diff
@@ -29,15 +29,15 @@ install:
       Write-Host "Installing GStreamer..." -ForegroundColor Cyan
       $msiPath = "$($env:USERPROFILE)\gstreamer-1.0-x86_64-1.14.4.msi"
       Write-Host "Downloading..."
-      (New-Object Net.WebClient).DownloadFile('https://s3-us-west-2.amazonaws.com/qgroundcontrol/dependencies/gstreamer-1.0-x86_64-1.14.4.msi', $msiPath)
+      (New-Object Net.WebClient).DownloadFile('https://gstreamer.freedesktop.org/data/pkg/windows/1.18.0/msvc/gstreamer-1.0-msvc-x86_64-1.18.0.msi', $msiPath)
```

Comment:
@DonLakeFlyer can you please upload these two installers (1.18.0) to s3?
https://gstreamer.freedesktop.org/data/pkg/windows/1.18.0/msvc/

Reply:
Done. Also FYI: The qmlsink code we are using has fixes in it which are specific to Windows. Not sure if that is related to why Windows works well, but other OS didn't alwy begin to work well.

Reply:
I moved these two installers to QGC S3 but this pull still references blue robotics S3.

Reply:
@DonLakeFlyer what are the addresses for such files ? We don't have access to QGC s3 console.

Reply:
Base path is `https://s3-us-west-2.amazonaws.com/qgroundcontrol/dependencies/` after that the filenames are the same.